composables/__tests__/useCustomerCreditCards.test.ts

```diff
@@ -0,0 +1,126 @@
+import { updatePaymentCard, deletePaymentCard, addPaymentCard } from "@/lib/gql/mutations"
+import { getPaymentCards } from "@/lib/gql/queries/checkout/getPaymentCards"
+import { useCustomerCreditCards } from "@/composables/useCustomerCreditCards"
```

Comment:
export from composables index file and import from @/composables

Reply:
done